### PR TITLE
Add new ECOMMERCE_COURSE_CATALOG_API_URL settings.

### PR DESCRIPTION
**Pull request by mjfrey**
New setting needed for ecommerce to communicate with the course catalog service.

### CODE REVIEW

**Comment by mjfrey** [comment]
@fredsmith @clintonb

File: playbooks/roles/ecommerce/defaults/main.yml [hunk 1]
```diff
@@ -125,6 +125,7 @@ ECOMMERCE_BROKER_HOST: '{{ ansible_default_ipv4.address }}'
 ECOMMERCE_BROKER_PORT: 5672
 ECOMMERCE_BROKER_URL: 'amqp://{{ ECOMMERCE_BROKER_USERNAME }}:{{ ECOMMERCE_BROKER_PASSWORD }}@{{ ECOMMERCE_BROKER_HOST }}:{{ ECOMMERCE_BROKER_PORT }}'
 
+ECOMMERCE_COURSE_CATALOG_URL: 'http://localhost:8008'
```
Comment (clintonb) [inline playbooks/roles/ecommerce/defaults/main.yml]: `ECOMMERCE_COURSE_CATALOG_API_URL: 'http://localhost:8008/api/v1/'`

File: playbooks/roles/ecommerce/defaults/main.yml [hunk 2]
```diff
@@ -134,6 +135,7 @@ ECOMMERCE_SERVICE_CONFIG:
   EDX_API_KEY: '{{ ECOMMERCE_EDX_API_KEY }}'
   OSCAR_FROM_EMAIL: '{{ ECOMMERCE_OSCAR_FROM_EMAIL }}'
 
+  COURSE_CATALOG_API_URL: '{{ ECOMMERCE_COURSE_CATALOG_API_URL }}/api/v1/'
```
Comment (clintonb) [inline playbooks/roles/ecommerce/defaults/main.yml]: `COURSE_CATALOG_API_URL: '{{ ECOMMERCE_COURSE_CATALOG_API_URL }}'`

**Comment by clintonb** [comment]
👍 (once comments addressed)

File: playbooks/roles/ecommerce/defaults/main.yml (hunk 2 above)
Comment (mjfrey) [inline playbooks/roles/ecommerce/defaults/main.yml]: @fredsmith suggested the original way so it is easier to substitute the real URL.

Comment (clintonb) [inline playbooks/roles/ecommerce/defaults/main.yml]: In that case, correct the name here to `ECOMMERCE_COURSE_CATALOG_URL`.

**Comment by fredsmith** [comment]
👍

**Pull request merged**